pkg/networkservice/core/next/tail_server.go

```diff
@@ -28,6 +30,10 @@ import (
 // to insure that we never call a method on a nil object
 type tailServer struct{}
 
+func newWrappedTailServer(wrapper ServerWrapper) networkservice.NetworkServiceServer {
```

Comment:
We should not modify next to fix this issue. 
`next` should not know anything about logging.